cyclonedx_py/parser/poetry.py

```diff
@@ -33,6 +35,7 @@ class PoetryParser(BaseParser):
 
     def __init__(
             self, poetry_lock_contents: str, use_purl_bom_ref: bool = False,
+            omit_category: Set[str] = set(),  # noqa: B006
```

Comment:
add this non-essential/optional argument after the `*`

Reply:
Collides with not having `noqa: B006`.

Reply:
Or make omit_category Optional[Set[str]]